.semaphore/semaphore.yml

```diff
@@ -9,6 +9,7 @@ agent:
 
 blocks:
   - name: Gradle build
+    dependencies: []
```

Comment:
These CI steps aren't dependent on one another, so I marked them as having no dependencies so that they can run in parallel.

Also, I noticed that `Gradle test` seems to also run `Gradle build`, so we might be able to combine them into just a single job.